src/sage/rings/ring.pyx

```diff
@@ -441,6 +441,7 @@ cdef class Ring(ParentWithGens):
             g = gens[0]
             if len(gens) == 1:
                 try:
+                    # TODO: Rewrite this properly
```

Comment:
What does this mean? If I saw this TODO i wouldn't know what to do properly.

Reply:
Oops I think that's a TODO for myself